.travis.yml

```diff
@@ -1,25 +1,29 @@
 ---
-  language: go
-  sudo: false
-  notifications:
-    email: false
-  matrix:
-    include:
-      - os: linux
-        go: 1.7.x
-      - os: linux
-        go: 1.8.x
-      - os: linux
-        go: tip
-      - os: osx
-        go: 1.8.x
-  install:
-    - echo "This is an override of the default install deps step in travis."
-  script:
-    - go build -v ./cmd/dep
-    - go vet $(go list ./... | grep -v vendor)
-    - test -z "$(gofmt -s -l . 2>&1 | grep -v vendor | tee /dev/stderr)"
-    - go test -race $(go list ./... | grep -v vendor)
-    - go build ./hack/licenseok
-    - find . -path ./vendor -prune -o -type f -name "*.go" -printf '%P\n' | xargs ./licenseok
-    - ./hack/validate-vendor.bash
+language: go
+sudo: false
+notifications:
+  email: false
+matrix:
+  include:
+    - os: linux
+      go: 1.7.x
+    - os: linux
+      go: 1.8.x
+    - os: linux
+      go: tip
+    - os: osx
+      go: 1.8.x
+install:
+  - echo "This is an override of the default install deps step in travis."
+before_script:
+  - PKGS=$(go list ./... | grep -v /vendor/)
+  - go get -v honnef.co/go/tools/cmd/staticcheck
```

Comment:
I added the `before_script` section so that fetching `staticcheck` and collecting the `$PKGS` list wouldn't pollute the logs. I also changed `grep -v vendor` to `grep -v /vendor/` to ensure that only the vendor _directory_ is filtered.